integration/simulation/network/network_utils.go

```diff
@@ -4,20 +4,21 @@ import (
 	"math/big"
 	"time"
 
+	config2 "github.com/obscuronet/obscuro-playground/go/obscuronode/config"
+
+	"github.com/obscuronet/obscuro-playground/go/ethclient/erc20contractlib"
+	"github.com/obscuronet/obscuro-playground/go/obscurocommon"
+	"github.com/obscuronet/obscuro-playground/go/obscuronode/enclave"
 	"github.com/obscuronet/obscuro-playground/integration"
+	p2p2 "github.com/obscuronet/obscuro-playground/integration/simulation/p2p"
```

Comment:
Again not a big fan of these "2"s. Like this one could be called sim_p2p or something perhaps so reader might have a sense of where it's coming from/why it's a different import

Reply:
Fixed (didn't add an underscore as Goland doesn't like it).